WORKSPACE

```diff
@@ -45,8 +49,15 @@ http_archive(
     sha256 = ENVOY_SHA256,
 )
 
-load("@envoy//bazel:repositories.bzl", "GO_VERSION", "envoy_dependencies")
+#local_repository(
+#     name = "envoy",
+#     path = LOCAL_ENVOY_PROJECT
+#)
```

Comment:
did stuff for building against a local envoy slip in here?   I think it's fine to keep it here commented out, but if you do, please add a few comments describing what it's for

Reply:
Leave here because I probably pasted these lines 10 times in the last 2 weeks... I haven't figure out if we can use bazel arg to switch between local or http_archive
Let me add the comment.